proto/protocolapis/astria/protocol/transaction/v1/action.proto

```diff
@@ -30,6 +31,10 @@ message Action {
     FeeAssetChange fee_asset_change = 53;
     FeeChange fee_change = 55;
     IbcSudoChange ibc_sudo_change = 56;
+
+    // Oracle actions are defined on 61-70
```

Comment:
Do you think we should start this range at 71 or even higher to leave plenty of room for other sudo actions in the future?

Reply:
yeah, doesn't hurt, i will change